examples/Q5_isingmodel-NumPy.py

```diff
@@ -0,0 +1,88 @@
+# Qubit Optimization for 3 qubit Ising model using  Pennylane default qubit ecosystem.
+
+# Ising models are used in Quantum Annealing to solve Quadratic Unconstrained Binary
+
+# Optimization problems with non-convex  cost functions. This example demonstrates how
+
+# Gradient descent optimizers can get stuck in local minima when using non-convex cost functions.
+
+# This example  uses and compares pennylane gradient descent optimizer with the optimizers
+
+# of PyTorch and and TensorFlow for this quantum system: see Q5_isingmodel.ipynb
```

Comment:
looks great! Might be a bit easier to read by removing the double line spacing here